build.properties.dist

```diff
@@ -14,7 +14,7 @@ platform.site.theme_default = ec_resp
 ecl.version = 0.18.0
```

Comment:
We need to update the ecl version, according the documentation of ec_europa.

Reply:
I make the modification.